mockserver/server.js

```diff
@@ -45,16 +45,6 @@ app.get('/v8/gfs/solar', (req, res, next) => {
   });
 });
 
-app.get('/v8/meta', (req, res) => {
-  res.json({
-    features: {
-      'feature-flag-1': true,
-      'flag-2': false,
-    },
-    callerLocation: [0, 0],
-  });
-});
```

Comment:
I would keep this in the mockserver anyway so it can be used for local testing by our contributors.

Reply:
It's in there! Just not twice, any more 🙈

Reply:
oh 😅

Reply:
I don't see it though 👀

Reply:
It's loading the JSON file directly if nothing is specifically defined - so a contributor can just change the JSON file and it will update :)

Reply:
oh (again), I thought it needed a registered route?